meta.yaml

```diff
@@ -0,0 +1,6 @@
+name: keda
+deployable_type: image
+lifecycle_status: development
```

Comment:
you can add `version: 0.1.0` if you want, so it will be easier to use instead of commit id